.github/workflows/tidelift.yml

```
@@ -15,6 +16,10 @@ on:
 permissions:
   contents: read
 
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
```

```suggestion
jobs:
```
I don't think this is necessary, as Tidelift Align only runs once a day.